tools/gulp/packages.ts

```diff
@@ -0,0 +1,9 @@
+import {BuildPackage, buildConfig} from 'material2-build-tools';
+import {join} from 'path';
+
+export const cdkPackage = new BuildPackage('cdk');
+export const materialPackage = new BuildPackage('material', [cdkPackage]);
+export const examplesPackage = new BuildPackage('material-examples', [materialPackage, cdkPackage]);
+
+// To avoid refactoring of the project the material package will map to the source path `lib/`.
+materialPackage.packageRoot = join(buildConfig.packagesDir, 'lib');
```

Comment:
I wanted to avoid something like that. It's really a special case that we want to overwrite the `packageRoot` here and it should not really be done through the build package.

Normally the build config is responsible for changing such things, but since this is really a special case I think this would be fine (it's just a single line; which is pretty clear IMO)